vm/rust/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+# Any upcoming updates that you aware of?
```

Comment:
Not really, I don't keep track of dependency updates apart from starknet related crates.